instrumentation/wicket-6.0/javaagent/wicket-6.0-javaagent.gradle

```diff
@@ -0,0 +1,21 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = 'org.apache.wicket'
+    module = 'wicket'
+    versions = "[6.0.0,]"
+    assertInverse = true
+  }
+}
+
+dependencies {
+  library group: 'org.apache.wicket', name: 'wicket', version: '8.0.0'
```

Comment:
Why not 6.0.0?

Reply:
Sure I can change it to 6.0.0 but it was released in 24-Aug-2012, 8.0.0 is from 16-May-2018. Our tests don't always run agains the earliest possible version, for example springmvc-3.1 runs agains 4.3.20, tomcat-7.0 runs agains 9.0.41, jaxrs-2.0-jersey-2.0 runs against 2.16. Should we test with the earliest possible version or something that has not yet started to fossilize?

Reply:
We should test earliest support version. If you have added support for Wicket 6+, then we should test with 6.0 :)

If some modules don't do that, then it is their fault and we should fix it.

Reply:
Or have clearly documented reason, why not.

Reply:
Changed supported version to 8.0, turns out I forgot to test with 6.0 and test didn't pass there.